README.md

```diff
@@ -1,2 +1,238 @@
+  ```bash  
+  Authorization: Bearer {token}
+  ```  
+  Replace **{token}** with the token provided to you.
+
+**Security Note**:
+ For security reasons, our API does not accept HTTP requests. All requests must be made over **HTTPS**.
+
+### Common Authentication Errors
+
+- **401 Unauthorized**: This error occurs when the token is missing or incorrect.
+
+### Security Recommendations
+
+**Use HTTPS**: Always use HTTPS to encrypt your requests.
+
+**Keep Your Token Secure**: Do not share your token publicly or embed it in publicly accessible code.
+
+**Monitor for Unauthorized Use**: If you suspect your token has been compromised, contact support immediately.
+
+## Request
+
+### Base URL
+
+- **API Base URL**: `TBD`
+
+### HTTP Method
+
+- **Method**: `POST`
+
+### Request Body
+
+- **Format**: `JSON`
+
+All relevant information for the request is sent as fields in the JSON body.
+
+### Fields Overview
+
+| Field Name |  Data Type | Description  | Requirement |  
+| ---- |  ---- | ----  | ---- |  
+| `url` | String | An absolute URL including a scheme (http:// or https://), or a MAID in case "trafficType": "MAID".  Note that setting this field to a MAID with a different traffic type will result in HTTP 400 response and using a URL when the traffic type is MAID will always result in a response with no contextual information provided. | **Required** |
+| `trafficType` | Enum | Possible values: <ul><li>`SITE` - normal page opened in a browser regardless of device</li><li>`MAID` - mobile app traffic where URL cannot be determined/shared and therefore the mobile app ID is provided for contextual classification</li><li>`CTV` - traffic from smart/internet-connected TV</li></ul> | **Optional (SITE by default)** |
+
+**Example JSON Payloads:**
+
+```json
+// For Site traffic
+{
+  "trafficType": "SITE",
+  "url": "https://www.sports.com"
+}
+
+// For App traffic
+{
+  "trafficType": "MAID",
+  "url": "wetter.at_app"
+}
+```
+
+### Headers & Authentication
+- **Headers:**
+  - `Content-Type: application/json`
+  - `Authorization: Bearer token_provided`
+- **Authentication:** Include the token in the Authorization header.
+
+### Examples & Code Snippets
+
+- **Successful Request Example** (Omitting `trafficType` - default **SITE**)  
+```bash
+curl -X POST "TBD" \
+  -H "Content-Type: application/json" \
+  -H "Authorization: Bearer ${CONTEXTUAL_API_TOKEN}" \
+  -d '{
+        "url": "https://www.sports.com"
+      }'
+```
+
+- **Successful Request Example** (Setting `trafficType` explicitly)  
+```bash
+curl -X POST "TBD" \
+  -H "Content-Type: application/json" \
+  -H "Authorization: Bearer ${CONTEXTUAL_API_TOKEN}" \
+  -d '{
+        "url": "https://www.sports.com",
+        "trafficType": "SITE"
+      }'
+```
+
+- **Error Request Example:**  (MAID traffic expected)
+```bash
+curl -X POST "TBD" \
+  -H "Content-Type: application/json" \
+  -H "Authorization: Bearer ${CONTEXTUAL_API_TOKEN}" \
+  -d '{
+        "url": "https://www.sports.com",
```

Comment:
Here we should use a real MAID in this example.
https://github.com/mbrtargeting/dmp-fluxv2/blob/main/prod-132/config/dmp-contextual-appname-segments.yaml

Reply:
Ah, sorry :facepalm: I should go have my eyes examined :laughing: